contracts/PinningManager.sol

```diff
@@ -191,16 +184,11 @@ contract PinningManager {
             uint256 toTransfer = (request.numberOfPeriodsDeposited - request.numberOfPeriodsWithdrawn).mul(request.chosenPrice);
             request.numberOfPeriodsWithdrawn = 0;
```

Comment:
Sorry can't comment on the proper line, but I am having bit doubt about how the `isPastCurrentEndTime` is calculated eq: `(request.startDate + request.numberOfPeriodsDeposited) * request.chosenPeriod `.

Why do you do `request.startDate + request.numberOfPeriodsDeposited` and then multiply it by period? Shouldn't it be: `request.startDate + (request.chosenPeriod * request.numberOfPeriodsDeposited)`?

Reply:
You are right

Reply:
Pushed the change